iroha-cpp/GrpcResponseHandler.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef GRPC_RESPONSE_HANDLER_HPP
+#define GRPC_RESPONSE_HANDLER_HPP
+
+#include "logger/logger_fwd.hpp"
+#include "logger/logger_manager_fwd.hpp"
+#include "QueryResponseHandler.hpp"
+#include "TransactionResponseHandler.hpp"
+
+
+namespace IROHA_CPP
+{
+
+
+namespace spdlog
+{
+    class logger;
+}
```

Comment:
The declaration should be not inside `namespace IROHA_CPP`

Reply:
Outdated comment.